documentation/api/io.strimzi.api.kafka.model.authentication.KafkaClientAuthenticationCustom.adoc

```diff
@@ -0,0 +1,90 @@
+To configure custom authentication, set the `type` property to `custom`.
+
+Custom authentication allows for any type of kafka supported authentication to be used.
```

Comment:
```suggestion
Custom authentication allows for any type of kafka-supported authentication to be used.
```

Reply:
`any type` sounds a bit weird. Should it be more about `custom type`?